benchmarks/vectorization_strategy_benchmark.py

```diff
@@ -356,64 +425,77 @@ def _parse_bounds(self, factor):
             return factor[0], factor[1]
         else:
             return type(factor)(0), factor
-
     @tf.function
     def call(self, inputs, training=True):
 
         augment = lambda: tf.map_fn(self._random_cutout, inputs)
         no_augment = lambda: inputs
         return tf.cond(tf.cast(training, tf.bool), augment, no_augment)
 
-    def _random_cutout(self, input):
-        center_x, center_y = self._compute_rectangle_position(input)
-        rectangle_height, rectangle_width = self._compute_rectangle_size(input)
-        rectangle_fill = self._compute_rectangle_fill(input)
-        input = fill_single_rectangle(
-            input,
-            center_x,
-            center_y,
-            rectangle_width,
-            rectangle_height,
-            rectangle_fill,
-        )
-        return input
+    def _random_cutout(self, inputs):
```

Comment:
Can you try to move the @tf.function from `call` to `_random_cutout`

Reply:
> Can you try to move the @tf.function from `call` to `_random_cutout`?

done

Reply:
Are you sure? I don't see this. Then try to use again the `vectorized_map` and check if you have bypassed the error.

Reply:
You can track this ticket https://github.com/tensorflow/tensorflow/issues/55340

Reply:
ok, got what you were trying to say, but looks like there needs to be some changes to be done in `fill_single_rectangle` as well. Working on it.......